pizzaserver-server/src/main/java/io/github/willqi/pizzaserver/server/network/BedrockClientSession.java

```diff
@@ -31,9 +34,10 @@
     private final Queue<BedrockPacket> queuedIncomingPackets = new ConcurrentLinkedQueue<>();
     private final Queue<BedrockPacket> queuedOutgoingPackets = new ConcurrentLinkedQueue<>();
 
-    public BedrockClientSession(BedrockServer server, RakNetServerSession rakNetServerSession) {
+    public BedrockClientSession(BedrockServer server, RakNetServerSession rakNetServerSession) throws IOException {
         this.server = server;
         this.serverSession = rakNetServerSession;
+        this.version = new V419MinecraftVersion(Server.getInstance());
```

Comment:
Check Discord but I'm not entirely sure if this is right. Maybe something else was causing the null issue. @-@